### DISCUSSION TITLE
IDE Helper in YOLO mode

### DISCUSSION THREAD

**Discussion by tommica** [lorisleiva/laravel-actions#281] [Ideas]
Would it be possible/interested for @lorisleiva to take ownership of the ide helper project, after the partial abandonment of the project by the original creator? https://github.com/Wulfheart/laravel-actions-ide-helper

**Comment by Wulfheart** [comment]
I have (don't know why exactly it happened) maintainer rights on this repository and I plan on taking over some maintenance duty over the next time. However, I don't want to deal with the ide-helper-package yet.
